Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/ConvertFragment.java

```diff
@@ -195,15 +199,14 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState)
 
   private void findViews()
   {
-    mFormatLayout = requireView().findViewById(R.id.format);
-    mFormatDropdown = requireView().findViewById(R.id.dropdown_format);
-    mBlockSizeLayout = requireView().findViewById(R.id.block_size);
-    mBlockSizeDropdown = requireView().findViewById(R.id.dropdown_block_size);
-    mCompressionLayout = requireView().findViewById(R.id.compression);
-    mCompressionDropdown = requireView().findViewById(R.id.dropdown_compression);
-    mCompressionLevelLayout = requireView().findViewById(R.id.compression_level);
-    mCompressionLevelDropdown = requireView().findViewById(R.id.dropdown_compression_level);
-    mRemoveJunkDataCheckbox = requireView().findViewById(R.id.checkbox_remove_junk_data);
+    mFormatDropdown = mBinding.dropdownFormat;
+    mBlockSizeLayout = mBinding.blockSize;
+    mBlockSizeDropdown = mBinding.dropdownBlockSize;
+    mCompressionLayout = mBinding.compression;
+    mCompressionDropdown = mBinding.dropdownCompression;
+    mCompressionLevelLayout = mBinding.compressionLevel;
+    mCompressionLevelDropdown = mBinding.dropdownCompressionLevel;
+    mRemoveJunkDataCheckbox = mBinding.checkboxRemoveJunkData;
```

Comment:
Could be removed. I do think these are used more often than the stuff I've commented on in previous files, but still, it's just a matter of typing `mBinding.`.